### ISSUE TITLE
DPL-101 - As Scott I need the new steps for for the RNA-seq part of the Combined LCM DNA / RNA-seq pipeline [ToR 12]

### ISSUE THREAD

**Issue by stevieing** [sanger/limber#789]
**User story**
As Scott I need the new steps for for the RNA-seq part of the Combined LCM DNA / RNA-seq pipeline

**Who are the primary contacts for this story**
e.g. John S (don't include surnames in public repos)

**Acceptance criteria**
To be considered successful the following steps need to be added:
- [ ] RNAseq cherrypick -> RNA XP and resuspension in dNTP/PolyT buffer
- [ ] RNA XP and resuspension in dNTP/PolyT buffer -> Denaturation
- [ ] Denaturation -> Reverse Transcription setup
- [ ] Reverse Transcription setup -> Reverse transcription
- [ ] Reverse transcription -> cDNA preamplification setup
- [ ] cDNA preamplification setup -> cDNA preamplification
- [ ] cDNA preamplification -> LB Cherrypick

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
[Combined LCM DNA- & RNA-seq.pdf](https://github.com/sanger/limber/files/6923118/Combined.LCM.DNA-.RNA-seq.pdf)

**Comment by stevieing** [comment]
[Combined LCM DNA- & RNA-seq.pdf](https://github.com/sanger/limber/files/6923139/Combined.LCM.DNA-.RNA-seq.pdf)

**Comment by JamesGlover** [comment]
Are these just the robot steps? I can't see any bed information in the PDF, we'll need that available to do anything here. We'll also need the information about which plates are being tracked in each part of the process. ( I notice the robot types are already available)

If these aren't just robot steps, and there's other new behaviour needed, then probably makes sense to pop that in a separate story.

**Comment by KatyTaylor** [comment]
Closing, as is in Done column.
